test/TestModule.cs

```diff
@@ -34,7 +36,23 @@ public TestModule()
                 var id = ctx.Request.Query.As<int>("id", 69);
                 await ctx.Response.WriteAsync($"Managed to parse default int {id}");
             });
-              
+
+            this.Post("/asstring", async ctx =>
```

Comment:
Have we got any tests that call this?

Reply:
I actually forgot to send it with this PR, sorry :)